specification/sbi_atee.adoc

```diff
@@ -13,6 +44,936 @@ running a VMM).
 The following operations must be implemented by the TSM ( _Formal input 
 parameters and return codes are TBD_ ).
 
+== Background
+
+Virtualization platforms are typically comprised of several components including
+platform firmware, host OS, VMM, and the actual payloads that run on them (typically
+in a VM). This model is well established, but the downside is that most platform
+components are in the TCB. This aspect is ill-suited for confidential compute
+workloads (Trusted Execution Environments) that strive to minimize the TCB footprint.
+
+We seek to alleviate this concern by introducing the notion of the a trusted broker for
+TEEs known as the TEE Security Manger (TSM). The idea is that the TSM has a minimal possible
+footprint, and acts as a trusted intermediary between the the untrusted platform components,
+and the TEE workloads that run on it. This is accomplished by having a clear seperation between
+the non-confidential and confidential portions of the workloads running on the platform. The exact
+hardware and software mechanisms used to guarantee the isolation are platform dependent, but
+confidential workloads have absolute execution, data integrity, and confidentiality protections
+from amy component in the non-confidential world. The responsibility of the TSM is to enforce 
+the guarantees accorded to TEE workloads. Given its minimal footprint, most of the functionality
+for actually running the payloads, like memory allocation, scheduling, interrupt handling, 
+virtual devices is left to the OS/VMM (which are outside the TCB).
+
+In this scheme, most resources like memory, (V)CPUs start off as traditional untrusted resources
+in the non-confidential world, and are transtioned to their confidential analogue via the TSM.
+Once the conversion process is complete, a confidential-resource cannot be reverted to the 
+non-confidential world unless it's is no longer assigned to a TVM.
```

Comment:
add a sentence "...via the TSM. Resources once transitioned to a confidential state, may be assigned to the TVM via the TSM."

and remove double negative to restate - "Once the conversion process is complete, a confidential resource may be reclaimed to the non-confidential world only when it is not assigned to a TVM. The assignment tracking is performed by the TSM."